### PR TITLE
Dont include imports for time:hour and time:minute dimensions

### PR DESCRIPTION
**Pull request by macobo**
Also include more information about import warnings in query results.

This mostly affects Stats API v2.

Documentation changes incoming.

### CODE REVIEW

File: lib/plausible/stats/query_result.ex [hunk 1]
```diff
@@ -70,7 +70,10 @@ defmodule Plausible.Stats.QueryResult do
 
   defp meta(query) do
     %{
-      warning:
+      imports_included: if(query.include.imports, do: query.include_imported, else: nil),
```
Comment (RobertJoonas) [inline lib/plausible/stats/query_result.ex:73]: I'm a little confused by this line here. I think it should always be a boolean. Wouldn't it work simply like this?

```suggestion
      imports_included: query.include.imports,
```

**Review by RobertJoonas — COMMENTED** [review]
The main graph doesn't display any warning or message about the main graph excluding imported data due to the applied interval. I'll fix that in this PR.

File: lib/plausible/stats/query_result.ex (hunk 1 above)
Comment (macobo) [reply lib/plausible/stats/query_result.ex:73]: Oops. This doesn't work and the tests luckily caught that. Intent is to signal whether imports were included successfully with this flag.

File: CHANGELOG.md [hunk 2]
```diff
@@ -20,6 +20,7 @@ All notable changes to this project will be documented in this file.
 - Testing framework for `/assets`
 
 ### Removed
+- Stop returning imported stats in timeseries queries for intervals shorter than `day`
```
Comment (RobertJoonas) [inline CHANGELOG.md:23]: This looks like a duplicate changelog entry for the same thing. Let's drop it.

```suggestion
```

**Review by RobertJoonas — APPROVED** [review]
Other than the duplicate changelog entry it's good to go! 👍

**Pull request merged**